core/server/worker/src/main/java/alluxio/worker/block/management/ManagementTaskCoordinator.java

```diff
@@ -181,12 +181,19 @@ private void runManagement() {
         LOG.debug("Running task of type:{}", currentTask.getClass().getSimpleName());
         // Run the current task on coordinator thread.
         try {
-          currentTask.run();
+          BlockManagementTaskResult result = currentTask.run();
+          LOG.debug("Management task: {} finished with result: {}",
+              currentTask.getClass().getSimpleName(), result);
```

Comment:
@calvinjia 
What do you think about making this log INFO level?

At the worst this would print every 10 seconds when;
- BackoffStrategy is DIRECTORY
- Tiers don't align
- AlignTask can't make progress (due to failures or heavy load)

I prefer this over crafting an admin report because the latter can't provide information for a full range without persistence at worker/master. I think this would be sufficient until at least we stabilize worker management infra.

Reply:
I think that's fine, we can also add a template to raise the log level of this class if it is too noisy.

Reply:
Cool. I don't think it will be noisy. I will watch for a while though.